src/strategies/erc20-balance-of-contract-multiplier/index.ts

```diff
@@ -0,0 +1,33 @@
+import { call } from '../../utils';
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+
+export const author = 'Phoenix';
```

Comment:
Should be github username
```suggestion
export const author = 'phoenix-keeper';
```